src/main/resources/data/mysticalworld/patchouli_books/world_guide/zh_cn/entries/cuisine/squid.json

```
@@ -0,0 +1,25 @@
+{
+  "name": "鱿鱼",
+  "icon": "mysticalworld:raw_squid",
+  "category": "cuisine",
+  "pages": [
+    {
+      "type": "spotlight",
+      "link_recipe": true,
+      "item": "mysticalworld:raw_squid",
+      "text": "可以通过宰杀$(l:creatures/squid)鱿鱼$()来掉落生鱿鱼须，可以直接生吃，但煮熟更好吃~"
+    },
+    {
+      "type": "smelting",
+      "recipe": "mysticalworld:cooked_squid",
+      "title": "烤鱿鱼",
+      "text": "为了用鱿鱼须做饭，您需要宰杀大量的鱿鱼，熟鱿鱼可以与其他成分相合成，带来更加壮观和神秘的体验！"

```

|Feedback Priority|
|---|
|⏳ Sand / Non-blocking but requires future consideration|

```suggestion
      "text": "想要尽情烹饪鱿鱼须，还得不断地宰杀鱿鱼，积累足够的食材才行，而熟鱿鱼又可以与其他材料相合成，最终，这一食材为我们带来了如此令人惊叹的神秘体验！"

```
本来是只想改ingredients的，但是越改越多

---

涉及到 Patchouli 指南书的翻译时，请检查一下在游戏内显示时会不会因为过长而超出页面。
我相信大佬是检查过了。

---

本页空间十分充足，最终选择权在你。